bin/GeosFpDriver.input

```diff
@@ -75,6 +75,7 @@ GEOS.fp.asm.inst3_3d_asm_Nv.YYYYMMDD_hhmm.V01.nc4
 PS,PV,QV,T
 
 ==> Local Raw Data Path
+# CHANGE the year and month folder in the following line
 /storage1/fs1/rvmartin/Active/GEOS-Chem-shared/MetFieldProcessing/GEOS_FP-raw/2021/03/
```

Comment:
I would recommend removing the year and month (`2021/03/` here) completely and instead have the raw files saved into GEOS-FP-raw. You can organize them into year/month folders later if archiving them. This would avoid having to update this path manually every time the date changes.